<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>712</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Packet Filter</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListView" name="rulesView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>191</width>
      <height>531</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::InternalMove</enum>
    </property>
    <property name="defaultDropAction">
     <enum>Qt::MoveAction</enum>
    </property>
    <property name="movement">
     <enum>QListView::Free</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="newRuleButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteRuleButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>550</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveApplyButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>550</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Save and apply</string>
    </property>
   </widget>
   <widget class="QPushButton" name="duplicateRuleButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>550</y>
      <width>92</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Duplicate</string>
    </property>
   </widget>
   <widget class="QDialogButtonBox" name="ruleEditButtonBox">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>550</y>
      <width>176</width>
      <height>27</height>
     </rect>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionNetfilter_output"/>
   </widget>
   <widget class="QMenu" name="menuApplication">
    <property name="title">
     <string>Application</string>
    </property>
    <addaction name="actionApply_modifications"/>
    <addaction name="actionReset"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuRule">
    <property name="title">
     <string>Rule</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionDuplicate"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionSave_rule"/>
    <addaction name="actionCancel_changes"/>
   </widget>
   <addaction name="menuApplication"/>
   <addaction name="menuRule"/>
   <addaction name="menuTools"/>
  </widget>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionApply_modifications">
   <property name="text">
    <string>Save and apply modifications</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Reset all modifications</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+R</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionSave_rule">
   <property name="text">
    <string>Save rule</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionDuplicate">
   <property name="text">
    <string>Duplicate</string>
   </property>
  </action>
  <action name="actionNetfilter_output">
   <property name="text">
    <string>Netfilter output</string>
   </property>
  </action>
  <action name="actionCancel_changes">
   <property name="text">
    <string>Cancel changes</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
